### PR TITLE
 REST API and Postman collection

### PR DESCRIPTION
**Pull request by ozhurova**
Added  REST API and Postman collection

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -12,1015 +12,1389 @@
...
       },
       "devDependencies": {
-        "nodemon": "^2.0.15"
-      }
-    },
-    "node_modules/@sindresorhus/is": {
-      "version": "0.14.0",
-      "resolved": "https://registry.npmjs.org/@sindresorhus/is/-/is-0.14.0.tgz",
-      "integrity": "sha512-9NET910DNaIPngYnLLPeg+Ogzqsi9uM4mSboU5y6p8S5DzMTVEsJZrawi+BoDNUVBa2DhJqQYUFvMDfgU062LQ==",
+        "eslint": "^8.10.0",
```
Comment (Morphine-medicine) [inline package-lock.json:15]: Package-lock.json should be removed from PR and added to gitignore

**Review by Morphine-medicine — COMMENTED** [review]
Looks good

File: package-lock.json (hunk 1 above)
Comment (ozhurova) [reply package-lock.json]: done

**Review by Morphine-medicine — COMMENTED** [review]
Looks good

**Pull request merged**